changelog/security/2022-06-09-ignition.md

```diff
@@ -0,0 +1 @@
+- ignition ([CVE-2022-1706](https://nvd.nist.gov/vuln/detail/CVE-2022-1706))
```

Comment:
Maybe a change entry would be good in addition?

Reply:
The change to enable it is done in this PR, so I think the operator notes link is very valuable here in case someone would have to opt out

Reply:
Ok, done. Thanks for the suggestion :)